### PR TITLE
Remove default value from batch size flags (Fixes #2204)

### LINKED ISSUE #2204: Default train batch size set to 1
- **Have I written custom code (as opposed to running examples on an unmodified clone of the repository)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 18.04
- **TensorFlow installed from (our builds, or upstream TensorFlow)**: Upstream tensorflow
- **TensorFlow version (use command below)**: 1.13.1 (v1.13.1-0-g6612da8951)
- **Python version**:  3.6.8
- **Bazel version (if compiling from source)**:
- **GCC/Compiler version (if compiling from source)**:
- **CUDA/cuDNN version**: 10.1/
- **GPU model and memory**: Gigabyte GTX 1070 (8gb)
- **Exact command to reproduce**: python3 DeepSpeech.py --train_files <train_file.csv> --dev_files <dev_file.csv> --test_files <test_file.csv>

If the `--train_batch_size` flag is not added for training, the default batch size is 1. This is a bug because the model will not train as well if the weights are updated this frequently. This also causes very inefficient use of gpus as most time is spent shuffling data across the PCIe connector and not training.

### PR DESCRIPTION
**Pull request by reuben**

### CODE REVIEW

**Comment by reuben** [comment]
Yeah. This is turning out to be more work than I expected. We'd have to make the flag optional at the abseil level and then check ourselves in config.py or something like that :/
